Makefile

```diff
@@ -51,15 +52,29 @@ _prepare-mobile: ##@prepare Install mobile platform dependencies and prepare wor
 	npm install
 
 $(STATUS_GO_IOS_ARCH):
-	curl -s -L \
+	curl --fail --silent --location \
 		"$(GITHUB_URL)/$(STATUS_GO_VER)/status-go-ios.zip" \
-		-o "$(STATUS_GO_IOS_ARCH)"
+		--output "$(STATUS_GO_IOS_ARCH)"; \
+	if [ $$? -ne 0 ]; then \
+		echo "Failed to download from GitHub, checking DigitalOcean Bucket..."; \
+    	curl --fail --silent --location \
```

Comment:
In this line, `||` is used instead of `if [ $$? -ne 0 ]; then` which might be seen as inconsistent. Also, if the second `curl` fails, does the whole target fail?

Reply:
You're right. Fixed.